www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb

```diff
@@ -126,7 +128,19 @@ do_download() {
 }
 ```
 
-After you have either specified your source in `pkg_source`, or overridden the **do_download()** callback, create a sha256 checksum for your source archive and enter it as the `pkg_shasum` value. The build script will verify this after it has downloaded the archive.
+The plan.ps1 equivelent would be:
+
+```powershell
+Function Invoke-Download { 
+  git clone https://github.com/chef/chef
+  pushd chef
+  git checkout $pkg_version
+  popd
+  Compress-Archive -Path chef/* -DestinationPath $HAB_CACHE_SRC_PATH/$pkg_name-$pkg_version.zip -Force
+  $script:pkg_shasum = (Get-FileHash -path $HAB_CACHE_SRC_PATH/$pkg_name-$pkg_version.zip -Algorithm SHA256).Hash.ToLower()
+}
+```
+After you have either specified your source in `pkg_source`, or overridden the **do_download()** or **Invoke-Download** callback, create a sha256 checksum for your source archive and enter it as the `pkg_shasum` value. The build script will verify this after it has downloaded the archive.
```

Comment:
I always cheat for this and run a build first, let it fail, and then paste in what Habitat told me it was expecting. 😉

Reply:
haha so do I! But this is what we have which is probably the better "official" guidance.

Reply:
Oh totally, I agree.